src/Command/Generate/EntityContentCommand.php

```diff
@@ -38,6 +38,13 @@ protected function configure()
             InputOption::VALUE_NONE,
             $this->trans('commands.generate.entity.content.options.is-translatable')
         );
+
+        $this->addOption(
+          'is-revisionable',
```

Comment:
Could you update this option as `revisionable` instead.

We are planning to update `is-translatable` as `translatable` only as well.